fuse/gocryptfs/Portfile

```diff
@@ -0,0 +1,143 @@
+                        lock    v0.4.1 \
+                        rmd160  5e94780d05150c19b5a97136811d8ffc1466f604 \
+                        sha256  86800e346c68162119293586010f32bcac20344293b97755ef4f11a1191b3c05 \
+                        size    8307 \
+                    github.com/kylelemons/godebug \
+                        lock    d65d576e9348 \
+                        rmd160  929cd615eff16a0c5ba2145b809b10016587a387 \
+                        sha256  9cbb2db613bd3a62e2e3f39776c690fb0d2c320d2aa5391868ad16d2de28c10c \
+                        size    14796 \
+                    github.com/jacobsa/reqtrace \
+                        lock    245c9e0234cb \
+                        rmd160  872beac620d3eebbc1a343d1d2ca65f2dc6778bd \
+                        sha256  e8ca3f8de13ce0f35c1cb2bb9c9be220c82f28a0a1ca80557e2afbbdf396a0c4 \
+                        size    7808 \
+                    github.com/jacobsa/ogletest \
+                        lock    80d50a735a11 \
+                        rmd160  d839d0d084bc62e25d23d65cee882b427b537c00 \
+                        sha256  8e9eef43d47e4fc12b6e614bd6c97d102eabec107eabe72493763d45325501fd \
+                        size    24200 \
+                    github.com/jacobsa/oglemock \
+                        lock    e94d794d06ff \
+                        rmd160  cee16f043ba80e59483d3bfadf783077f8239110 \
+                        sha256  5474b9f5228d6eda866293feb9c07729a55195fabd2cd77cd1c30e5291830ad9 \
+                        size    35506 \
+                    github.com/jacobsa/oglematchers \
+                        lock    141901ea67cd \
+                        rmd160  5c5dd8416ee4a236632eaa4fc1a1e5ce737eee45 \
+                        sha256  4a45ac5d5b5c15af45761f15e3e14f4739b1cd79cd9493ddcd4744f65edc4b52 \
+                        size    43827 \
+                    github.com/jacobsa/crypto \
+                        lock    9f44e2d11115 \
+                        rmd160  0cbf3ba64ed7e4a65ad9e5083efe103deb2137f7 \
+                        sha256  8c230508b648a74dfecf86015965bd89c78b98a2e5988ee7ab538940db078390 \
+                        size    3653945 \
+                    github.com/hanwen/go-fuse \
+                        lock    ${gitversionfuse} \
+                        rmd160  60efe988c58e58c307bf86e3271ea7c419cd19eb \
+                        sha256  6fd20e10d9b30020f1cdee312c2906f85904894858bcd9248cf36e5912f2022f \
+                        size    204291 \
+                    github.com/davecgh/go-spew \
+                        lock    v1.1.0 \
+                        rmd160  0303eae19a01f38fe314921fd965e4d09b9ef3ad \
+                        sha256  810a597004388d68bb92d8aa612375419ba1080dd5fc2c66dd41b58f0ba4442c \
+                        size    42348
+
+set builddate       [exec date +"%Y-%m-%d"]
+set ldflags         "-X \"main.GitVersion=${version}\" -X \"main.GitVersionFuse=${gitversionfuse}\" -X \"main.BuildDate=${builddate}\""
```

Comment:
Please do not hardcode a build timestamp into the binary. This goes against the requirements of [reproducible builds](https://reproducible-builds.org/docs/timestamps/). Can we simply leave this out or do we need to provide a value for `main.BuildDate`?